ThePortugueseMan.HabitTracker/Habit_DataBase/DataBaseCommands.cs

```diff
@@ -0,0 +1,394 @@
+﻿namespace DataBaseLibrary;
+using Microsoft.Data.Sqlite;
+using System;
+using System.Globalization;
+
+
+public class DataBaseCommands
+{
+    static string connectionString = @"Data Source=habit-Tracker.db";
+    static string s_MainTableName = "HabitsTable";
+
+    //if the main table doesn't exist, it's created
+    public void Initialization(string? mainTableName)
+    {
+        using (var connection = new SqliteConnection(connectionString))
+        {
+            connection.Open();
+            var tableCmd = connection.CreateCommand();
+
+            tableCmd.CommandText =
+                @$"CREATE TABLE IF NOT EXISTS {mainTableName}" +
+                    "(Id INTEGER PRIMARY KEY AUTOINCREMENT," +
+                    "HabitTableName TEXT," +
+                    "HabitUnit TEXT)";
+
+            tableCmd.ExecuteNonQuery();
+
+            connection.Close();
+        }
+    }
+    //creates a new subtable represeting an habit - each entry has a string date and an int quantity
```

Comment:
Please check the necessity of this comment. 😊

Reply:
Made the function name more clear and removed the comment